bazel/foreign_cc/BUILD

```diff
@@ -25,13 +25,17 @@ envoy_cmake_external(
     cache_entries = {
         "EVENT__DISABLE_OPENSSL": "on",
         "EVENT__DISABLE_REGRESS": "on",
+        "EVENT__LIBRARY_TYPE": "STATIC",
         "CMAKE_BUILD_TYPE": "Release",
     },
     copy_pdb = True,
     lib_source = "@com_github_libevent_libevent//:all",
     static_libraries = select({
         "//bazel:windows_x86_64": ["event.lib"],
-        "//conditions:default": ["libevent.a"],
+        "//conditions:default": [
+            "libevent.a",
+            "libevent_pthreads.a",
```

Comment:
Which change lead ninja generate separate archive for `libevent_pthreads`?

Reply:
Not sure, when I built I got a link error on `evthread_use_pthreads()`, which the docs say we should link with `libevent_pthreads` to find.